API/Repositories/Implementations/OAuthClientRepository.cs

```diff
@@ -24,4 +20,20 @@ public async Task<bool> ValidateAsync(int clientId, string clientSecret)
 
         return match != null;
     }
+
+    public async Task<OAuthClient?> SetRatelimitOverridesAsync(int clientId, RateLimitOverrides rateLimitOverrides)
+    {
+        OAuthClient? match = await _context.OAuthClients.FirstOrDefaultAsync(x => x.Id == clientId);
```

Comment:
```suggestion
        OAuthClient? client = await GetAsync(clientId);
```

Use built in `GetAsync`